src/server/game/AuctionHouse/AuctionHouseListing.cpp

```diff
@@ -129,6 +129,10 @@ AuctionListingEvent::AuctionListingEvent(Player* player, uint32 faction) :
 {
 }
 
+AuctionListingEvent::~AuctionListingEvent()
```

Comment:
It is in cpp file, since i moved all definitions out of header this should be ok